### PR TITLE
Renamed embedded csp header to requires csp

### PR DESCRIPTION
**Pull request by andypaicu**
Renamed the HTTP header from 'embedding-csp' to 'required-csp'

### CODE REVIEW

File: embedded/index.src.html [hunk 1]
```diff
@@ -173,12 +173,10 @@
       element. This is described in more detail in [[#csp-attribute]].
       
   2.  That attribute's value will be sent along with any <a>navigation request</a> that targets
-      the <{iframe}>'s <a>nested browsing context</a> in an <a http-header>`Embedding-CSP`</a>
-      HTTP request header. This header is described in more detail in [[#embedding-csp-header]].
+      the <{iframe}>'s <a>nested browsing context</a> in an <a http-header>`Required-CSP`</a>
```
Comment (mikewest) [inline embedded/index.src.html]: s/an/a/

File: embedded/index.src.html [hunk 2]
```diff
@@ -277,20 +275,20 @@
 
   ISSUE: Upstream this to all the HTMLs.
  
-  The `Embedding-CSP` HTTP Request Header {#embedding-csp-header}
+  The `Required-CSP` HTTP Request Header {#required-csp-header}
   ---------------------------------------
 
   In order to ensure that the embedded resource can decide whether or not it is willing to adhere to
   the embedder's requirements, the policy expressed in an <{iframe}>'s <{iframe/csp}> attribute is
   communicated along with affected <a>navigation requests</a> via an
```
Comment (mikewest) [inline embedded/index.src.html]: s/an/a/

**Comment by mikewest** [comment]
Thanks!

**Comment by mikewest** [comment]
@wseltzer: Andy's working with me at Google. Given that, I've landed this despite the bot's IPR protestations, and have pointed Andy at the relevant forms for joining the WG.

**Comment by wseltzer** [comment]
and he's now joined, thanks!

**Pull request merged**